system-variables.md

```diff
@@ -3702,7 +3702,7 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Type: Enumeration
-- Default value: `pessimistic`
+- Default value: ""
```

Comment:
The default is "", but also "" means pessimistic. It's an anti-feature caused by the PM team not being able to figure out upgrade-downgrade rules. i.e. since an inplace upgrade from 2.1 is supported even to the latest version, it will forever be confusing. See: https://github.com/pingcap/tidb/pull/25009

It should say in the text that values of "" also mean pessimistic.

Reply:
```suggestion
- Default value: `pessimistic`
```